tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -76,3 +82,78 @@ describe('OptionsListUtils', () => {
         await measureFunction(() => OptionsListUtils.getMemberInviteOptions(personalDetails, mockedBetas, SEARCH_VALUE));
     });
 });
+
+/* formatSectionsFromSearchTerm performance tests */
+describe('formatSectionsFromSearchTerm', () => {
+    test('[OptionsListUtils] formatSectionsFromSearchTerm with an empty search term and a large number of selectedOptions', async () => {
+        const selectedOptions: ReportUtils.OptionData[] = Array.from({length: SELECTED_OPTIONS_COUNT}, (_, i) => ({
```

Comment:
lets use `createCollection` and we can create something like `createRandomPolicy` to get the object we wanted.
more details here https://github.com/Expensify/App/blob/6cdf088114314447c546467025d465568ef8f690/tests/README.md#mocking-collections--collection-items